Monika-After-Story/MonikaModDev
# New-O31

# New O31

## :crab: RIN IS FIXED 🦀 
### New ACS (NOTE: These are NOT selectable):
- `mas_acs_rin_bows_front`
- `mas_acs_rin_bows_back`
- `mas_acs_rin_ears`

## TODO:
- [x] o31 cleanup in session to re-unlock the bg selector
- [x] thumbnail for rin

## Testing:
- Make sure Rin looks good
- Make sure CGs work again
- Make sure spritepack based outfit greetings work too
- Verify that if you're in another background prior to loading in on o31 (or get the o31 relaunch), loading in results in you back in spaceroom, with the bg selector locked
